antora/ec-policies-antora-extension/index.js

```diff
@@ -6,6 +6,18 @@ const opa = import("@zregvart/opa-inspect")
 // Helpers for handlebars templates
 const hbsHelpers = {
 
+  // Determines if value is in array
+  isInArray: (value, array) => {
+    if (Array.isArray(array)) {
+      return array.includes(value);
+    } else if (typeof array === 'string') {
+      // Parse the array string into an actual array
+      const parsedArray = JSON.parse(array);
+      return parsedArray.includes(value);
+    }
+    return false;
+  },
+
```

Comment:
Since we're in plain javascript, I guess here this would work here:

    isBuiltin: (value) => {
      return hbsHelpers.isInArray(value, ["builtin_attestation", "builtin_image"])
    }

Which I think provides another way to ditch the need for json.Parse.

Reply:
Alternative version:

```
isBuiltin: (val) => {
  return val.startsWith("builtin_")
},
```